vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/model/TestCaseStatus.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.zephyr.model;
+
+public enum TestCaseStatus
+{
+    FAILED(2),
+    BROKEN(12),
+    PENDING(12),
+    UNKNOWN(10),
+    SKIPPED(-1),
+    PASSED(1);
```

Comment:
are ID-s static and applicable for any Zephyr instance? or are they dynamic?

Reply:
I think dynamic https://getzephyr.docs.apiary.io/#reference/utilresource/get-execution-statuses-priorities-components-labels/get-execution-statuses,-priorities,-components,-labels

Reply:
for meta also ?

I thought it should be looks like `@testCaseId test-1 test-2`

Reply:
I think we need to collect ID-s at runtime using the mentioned endpoint